app/controllers/result_controller.rb

```diff
@@ -1,4 +1,3 @@
 class ResultController < ApplicationController
-  def index
-  end
+  def index; end
```

Comment:
Oh, what was the error? For me it complains when there is a blank line in the middle, but otherwise it passes :man_shrugging: